### ISSUE TITLE
Add Component Type - Plugin

### ISSUE THREAD

**Issue by kencinder** [XDjackieXD/PackUpdate#34]
I still make use of this tool, it's simple and works great. I use it for keeping my servers and clients updated.

Would be nice if a component type for plugins could be added, so plugins could be added using the tool as well.

**Comment by Kilobyte22** [comment]
I'm assuming you mean server side plugins, for Bukkit-Compatible servers?

That should be pretty easy to add. I'll see about that later when i am not about to go to bed :D (and probably forget again - but hopefully not)

**Comment by kencinder** [comment]
Yes, bukkit...but also the likes of Spongeforge is going the way of separating plugins into their own directory as opposed to having been in `mods`
